Firestore/Swift/Source/Helper/PipelineHelper.swift

```diff
@@ -0,0 +1,51 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+enum Helper {
```

Comment:
Why an enum instead of a struct or class? Is this a swift convention?

Reply:
Since Helper class only contains "static" methods, it should not be construct by itself. Since it is used internally, if we have other use case in the future, we can change it to struct.